src/lib.rs

```diff
@@ -4,6 +4,7 @@
 #![allow(clippy::match_bool)]
 #![feature(ptr_wrapping_offset_from)]
 #![feature(drain_filter)]
+#![feature(try_trait)]
```

Comment:
lets not add new unstable features please, with the latest version we should be able to actually move to a stable compiler by default

Reply:
I have made a separate commit which replaces all `?` with a new `job_try!` macro.